.github/workflows/build-push-stolostron.yaml

```diff
@@ -0,0 +1,30 @@
+name: build and push to quay
+
+on:
+  workflow_dispatch:
```

Comment:
Sorry, I missed this the first time. We don't want this workflow to be run on-demand.

```suggestion
```